client-side/src/stories/DateTime.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { DemoContainer } from '@mui/x-date-pickers/internals/demo';
+import { LocalizationProvider } from '@mui/x-date-pickers-pro/LocalizationProvider';
+import { AdapterDayjs } from '@mui/x-date-pickers-pro/AdapterDayjs';
+import { DateTimeRangePicker } from '@mui/x-date-pickers-pro/DateTimeRangePicker';
```

Comment:
put all imports from '@mui/x-date-pickers-pro' in one line
something like import {DemoContainer , LocalizationProvider , ...} from '@mui/x-date-pickers-pro'